tensorflow/examples/android/src/org/tensorflow/demo/CameraActivity.java

```diff
@@ -62,6 +87,103 @@ protected void onCreate(final Bundle savedInstanceState) {
     }
   }
 
+  /*
```

Comment:
There is a vim plugin for andriod studio which which is fantastic.  But I still miss vim sometimes.